.github/workflows/publish.yaml

```diff
@@ -0,0 +1,26 @@
+name: Publish to gh-pages
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 8
```

Comment:
Bump to LTS (v10) or v12?

Reply:
The [docusaurus docs](https://docusaurus.io/docs/en/publishing#automating-deployments-using-continuous-integration) all show using Node 8

Reply:
i suspect that's just stale - not for any good reason